visidata/loaders/tabulate.py

```diff
@@ -0,0 +1,19 @@
+import functools
+from visidata import vd, Progress
+
+try:
+    import tabulate
+    for fmt in tabulate_formats:
+        def save_table(path, *sheets):
+            import tabulate
+
+            with path.open_text(mode='w') as fp:
+                for vs in sheets:
+                    fp.write(tabulate.tabulate(
+                        vs.itervals(*vs.visibleCols, format=True),
+                        headers=[ col.name for col in vs.visibleCols ],
+                        tablefmt=fmt))
```

Comment:
@saulpw It looks like you are hitting this issue: https://bugs.python.org/issue13652

All the savers end up saving the same format, the last one listed in tabulate_formats.